odk1-src/form-logic.rst

```diff
@@ -482,12 +482,38 @@ Complex example
     integer, b, b =,
     calculate, a_plus_b, ,"if(${a} != '' and ${b} != '', ${a} + ${b}, '')"
     note, display_sum, a + b = ${a_plus_b}, 	
-  
-    
+
+.. _groups:
+
+Groups of questions
+====================
+
+To group, questions use the :tc:`begin_group...end_group` syntax.
+
+.. rubric:: XLSForm (Question group)
+
+.. csv-table:: survey
+  :header: type, name, label
+
+  begin_group, my_group, My text widgets
+  text, question_1, Text widget 1
+  text, question_2, These questions will both be grouped together
+  end_group, ,
+
+If given a :th:`label`, groups will be visible in the form path to help orient the user
+(e.g. :guilabel:`My text widgets > Text widget 1`).
+
+If given a :th:`label` and also a :th:`ref`, groups will be visible as clickable items in the jump menu:
```

Comment:
Some language here (or up at top of of the group section) about logical groups and visual groups would nice. Maybe in a note (Search the rst for `note::` for an example) that says groups without labels can be helpful for grouping questions in a way that isn't shown to the user. This technique be helpful for internal organization of the form. You should say it better than I just said it. 😂

Rather than say `ref`, which is likely to confuse, I'd put a warning (Search the rst for `warning::` for an example) that says if you use Build v0.3.4 or earlier, your group will not show as a group in the jump menu).